index.html

```diff
@@ -14,36 +15,2214 @@
+	<tr>
+		<td>2012-10</td>
+		<td>7.8</td>
+		<td>40.7</td>
+	</tr>
+	<tr>
+		<td>2012-11</td>
+		<td>7.7</td>
+		<td>40.1</td>
+	</tr>
+	<tr>
+		<td>2012-12</td>
+		<td>7.9</td>
+		<td>38.8</td>
+	</tr>
+	<tr>
+		<td>2013-01</td>
+		<td>8.0</td>
+		<td>37.5</td>
+	</tr>
+	<tr>
+		<td>2013-02</td>
+		<td>7.7</td>
+		<td>39.2</td>
+	</tr>
+	<tr>
+		<td>2013-03</td>
+		<td>7.5</td>
+		<td>39.0</td>
+	</tr>
+	<tr>
+		<td>2013-04</td>
+		<td>7.6</td>
+		<td>37.2</td>
+	</tr>
+	<tr>
+		<td>2013-05</td>
+		<td>7.5</td>
+		<td>37.5</td>
+	</tr>
+	<tr>
+		<td>2013-06</td>
+		<td>7.5</td>
+		<td>37.2</td>
+	</tr>
+	<tr>
+		<td>2013-07</td>
+		<td>7.3</td>
+		<td>37.9</td>
+	</tr>
+	<tr>
+		<td>2013-08</td>
+		<td>7.3</td>
+		<td>38.4</td>
+	</tr>
+	<tr>
+		<td>2013-09</td>
+		<td>7.3</td>
+		<td>36.9</td>
+	</tr>
+	<tr>
+		<td>2013-10</td>
+		<td>7.2</td>
+		<td>35.7</td>
+	</tr>
+	<tr>
+		<td>2013-11</td>
+		<td>6.9</td>
+		<td>37.8</td>
+	</tr>
+	<tr>
+		<td>2013-12</td>
+		<td>6.7</td>
+		<td>37.2</td>
+	</tr>
+	<tr>
+		<td>2014-01</td>
+		<td>6.6</td>
+		<td>35.3</td>
+	</tr>
+	<tr>
+		<td>2014-02</td>
+		<td>6.7</td>
+		<td>36.5</td>
+	</tr>
+	<tr>
+		<td>2014-03</td>
+		<td>6.7</td>
+		<td>35.4</td>
+	</tr>
+	<tr>
+		<td>2014-04</td>
+		<td>6.2</td>
+		<td>34.9</td>
+	</tr>
+	<tr>
+		<td>2014-05</td>
+		<td>6.2</td>
+		<td>34.5</td>
+	</tr>
+	<tr>
+		<td>2014-06</td>
+		<td>6.1</td>
+		<td>32.9</td>
+	</tr>
+	<tr>
+		<td>2014-07</td>
+		<td>6.2</td>
+		<td>33.3</td>
+	</tr>
+	<tr>
+		<td>2014-08</td>
+		<td>6.2</td>
+		<td>31.2</td>
+	</tr>
+	<tr>
+		<td>2014-09</td>
+		<td>6.0</td>
+		<td>31.9</td>
+	</tr>
+	<tr>
+		<td>2014-10</td>
+		<td>5.7</td>
+		<td>31.8</td>
+	</tr>
+	<tr>
+		<td>2014-11</td>
+		<td>5.8</td>
+		<td>31.1</td>
+	</tr>
+	<tr>
+		<td>2014-12</td>
+		<td>5.6</td>
+		<td>31.8</td>
+	</tr>
+	<tr>
+		<td>2015-01</td>
+		<td>5.7</td>
+		<td>31.3</td>
+	</tr>
+	<tr>
+		<td>2015-02</td>
+		<td>5.5</td>
+		<td>30.8</td>
+	</tr>
+	<tr>
+		<td>2015-03</td>
+		<td>5.5</td>
+		<td>29.5</td>
+	</tr>
+	<tr>
+		<td>2015-04</td>
+		<td>5.4</td>
+		<td>28.7</td>
+	</tr>
+	<tr>
+		<td>2015-05</td>
+		<td>5.5</td>
+		<td>28.7</td>
+	</tr>
+	<tr>
+		<td>2015-06</td>
+		<td>5.3</td>
+		<td>25.9</td>
+	</tr>
+	<tr>
+		<td>2015-07</td>
+		<td>5.3</td>
+		<td>27.1</td>
+	</tr>
+	<tr>
+		<td>2015-08</td>
+		<td>5.1</td>
+		<td>27.7</td>
+	</tr>
+	<tr>
+		<td>2015-09</td>
+		<td>5.1</td>
+		<td>26.6</td>
+	</tr>
+	<tr>
+		<td>2015-10</td>
+		<td>5.0</td>
+		<td>26.7</td>
+	</tr>
+	<tr>
+		<td>2015-11</td>
+		<td>5.0</td>
+		<td>25.7</td>
+	</tr>
+	<tr>
+		<td>2015-12</td>
+		<td>5.0</td>
+		<td>26.3</td>
+	</tr>
+	<tr>
+		<td>2016-01</td>
+		<td>4.9</td>
+		<td>26.9</td>
+	</tr>
+	<tr>
+		<td>2016-02</td>
+		<td>4.9</td>
+		<td>27.7</td>
+	</tr>
+	<tr>
+		<td>2016-03</td>
+		<td>5.0</td>
+		<td>27.6</td>
+	</tr>
+	<tr>
+		<td>2016-04</td>
+		<td>5.0</td>
+		<td>25.7</td>
+	</tr>
+	<tr>
+		<td>2016-05</td>
+		<td>4.7</td>
+		<td>25.1</td>
+	</tr>
+
```

Comment:
Wheee! That was a fun bunch of scrolling. Won't it be great when we can load data dynamically?